glib/src/auto/enums.rs

```diff
@@ -284,20 +284,28 @@ impl FromGlib<ffi::GDateMonth> for DateMonth {
 #[non_exhaustive]
 #[doc(alias = "GDateWeekday")]
 pub enum DateWeekday {
+    // rustdoc-stripper-ignore-next-stop
```

Comment:
No please don't add these; the CI complains about them due to an error in rustdoc-stripper (that no one took the time to debug yet)

Reply:
Sorry, I removed them again